src/mono/wasm/Wasm.Build.Tests/IcuTests.cs

```diff
@@ -232,21 +60,17 @@ public void FullIcuFromRuntimePackWithInvariant(BuildArgs buildArgs, bool invari
     }
 
     [Theory]
-    [MemberData(nameof(FullIcuWithICustomIcuTestData), parameters: new object[] { false, false, RunHost.NodeJS | RunHost.Chrome })] 
-    [MemberData(nameof(FullIcuWithICustomIcuTestData), parameters: new object[] { true, true, RunHost.NodeJS | RunHost.Chrome })]
-    public void FullIcuFromRuntimePackOrCustomIcu(BuildArgs buildArgs, bool fullIcu, bool useCustomFile, RunHost host, string id)
+    [MemberData(nameof(FullIcuWithICustomIcuTestData), parameters: new object[] { false, RunHost.NodeJS | RunHost.Chrome })]
```

Comment:
We should be checking the case where `$(WasmIncludeFullIcuData)` is unset, so the default. So, maybe test the case of unset, and non-default value. This can be in a follow up PR.